### PR TITLE
allow parsing from decompressed lzma

### PR DESCRIPTION
**Pull request by Liam-DeVoe**
to support https://github.com/circleguard/circlecore/pull/142

### CODE REVIEW

File: circleparse/replay.py [hunk 1]
```diff
@@ -144,8 +144,12 @@ def parse_play_data(self, replay_data):
             else:
                 del self.play_data[-1]
 
-    def data_from_lmza(self, lzma_string):
-        datastring = lzma.decompress(lzma_string, format=lzma.FORMAT_AUTO).decode('ascii')[:-1]
+    def data_from_lmza(self, lzma_string, decompressed_lzma):
+        if decompressed_lzma:
+            # replay data is already decompressed and decoded
+            datastring = lzma_string[:-1]
```
Comment (InvisibleSymbol) [inline circleparse/replay.py:151]: any reason why you are dropping the last char?

Comment (Liam-DeVoe) [reply circleparse/replay.py:151]: that's a comma and splitting doesn't work unless we remove it. Same thing happens normally

Comment (InvisibleSymbol) [reply circleparse/replay.py:151]: can we add a comment so it's clear why we remove it?

**Review by InvisibleSymbol — APPROVED** [review]
lgtm

**Pull request merged**